Mlem/App/Models/Settings/Options/PostSize.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  PostSize.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-05-19.
+//
+
+import Foundation
+import SwiftUI
+
+enum PostSize: String, CaseIterable {
+    case compact, tile, headline, large
```

Comment:
Considering how this'll work with the rest of the app, considering that tiled posts can only be shown in feeds - I reckon we should remove `tile` from this enum and have a `experiments.tiledPosts` boolean `AppStorage` value to control whether tiled posts are on or not.

Reply:
Sure, that works. It makes a couple switch statements a bit ungainly but maps more accurately to the behavior.